data/src/error_analysis/README.md

```diff
@@ -0,0 +1,61 @@
+Error analysis tool for grapheme to phoneme (g2p) conversion
+============================================================
+
+This tool performs a fine-grained error analysis of a G2P model. It reports a
+performance matrix of a test output generated by a G2P model.
+
+Under this error analysis process, each test record generated from a G2P model
+is evaluated under two criteria. ⋅⋅\* if the hypothesized form matches the
+corpus prediction or not. ⋅⋅\* if the hypothesized form adheres to grammatical
+rules of the language.
+
+The output of the tool is a two dimentional matrix as shown below-
+
+               | CG Match  |   CG Not Match |
+---------------|-----------+----------------|
+Pron Match     |  79.34    |      05.01     |
+Pron Not Match |  13.15    |      02.50     |
+
+Here the numbers represents percentage of test records falling under X-section
+of each category.
+
+Prerequisites
+-------------
+
+### Library
+
+install pynini
+install prettytable
```

Comment:
You could put this one in `data/requirements.txt` at least.

Reply:
`data/requirements.txt` has the following content:

``` 
ipapy>=0.0.9.0
regex>=2019.12.9
requests
requests-html
wikipron>=1.0.0
```

I am adding these two lines in the same file.
```
1. install pynini
conda install -c conda-forge pynini

2. install prettytable
pip install prettytable
```

Reply:
Great, thanks.

Reply:
updated.